tracer/src/Datadog.Tracer.Native/debugger_constants.h

```diff
@@ -33,7 +34,8 @@ const WSTRING skip_assemblies[]{WStr("mscorlib"),
                                 WStr("Anonymously Hosted DynamicMethods Assembly"),
                                 WStr("Datadog.AutoInstrumentation.ManagedLoader"),
                                 WStr("ISymWrapper"),
-                                WStr("Datadog.Trace")};
+                                WStr("Datadog.Trace"),
+                                WStr("MSBuild")};
```

Comment:
Before approving this I would like to test if skipping `MSBuild` affects CI Visibility.

Reply:
I'm skipping `MSBuild` only for the Debugger's Exploration Tests, it has nothing to do with other products (note the `debugger_constants.h`).
Is that makes sense?

Reply:
Oh you're right, I missed the name of the file.